src/main/msp/msp.c

```diff
@@ -2192,6 +2193,7 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
                     mac->linkedTo = findBoxByPermanentId(i)->boxId;
                 }
                 rcControlsInit();
+                pinioBoxTaskControl();
```

Comment:
It would be cleaner if this was called at the end of the function that parses the box activation conditions (`rcControlsInit()`)- then we can be sure it will be updated every time these are changed. Then it can also be removed from `tasksInit()`.

Reply:
I pushed an update but hold on it - need some more testing. I'm not sure I can remove the initialization in `tasksInit()` based on the order that things happen.

Reply:
Yeah, can't get around having it in `tasksInit()`. `rcControlsInit()` gets called early during `activateConfig` after reading the eeprom. Near the very end of `init()` `tasksInit()` is called which calls `schedulerInit()` which first thing clears the task queue. So this wipes out the `TASK_PINIOBOX` that was enabled earlier.